manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "XTranslate",
   "description": "__MSG_description__",
-  "version": "2.76.2",
+  "version": "2.76.3",
```

Comment:
Please rollback. There is no need to maintain new version for you _(it might be different when it goes to production)._